winterfell/webapp/src/styles/signin.styl

```diff
@@ -1,97 +1,216 @@
 @import "standards.styl"
 
+backgroundUrl = '../icons/veteran-dog-bg.jpg'
```

Comment:
I noticed this image is ~ 3 MB, makes the page slow even serving locally, not sure it'll be suitable for production, is there anything else we can use instead?